components/nav.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="header">
+    <div class="title">
+      <a class="reglink" href="/landingpage">
```

Comment:
Almost forgot! Nuxt uses a special routing logic. Instead of using `a` tags, take a look at the [NuxtLink](https://v3.nuxtjs.org/api/components/nuxt-link#nuxtlink) docs.

Reply:
It's worth noting, if we wanted to link an external page, we'd still use `a` tags but for anything internal we want to use  `NuxtLink`.

Also! The landing page @dogboots is working on will be at `/` rather. than `/landingpage`.